### PR TITLE
specified main()'s return type

### PR DESCRIPTION
**Pull request by bcko**
https://www.dartlang.org/guides/language/language-tour#the-main-function

### CODE REVIEW

File: bin/dart2_fix.dart [hunk 1]
```diff
@@ -7,7 +7,7 @@ import 'dart:io';
 import 'package:dart2_fix/src/dart_fix.dart';
 import 'package:dart2_fix/src/model.dart';
 
-main(List<String> args) async {
+void main(List<String> args) async {
```
Comment (devoncarew) [inline bin/dart2_fix.dart]: This should be `Future` :) (because its an async method)

Comment (bcko) [reply bin/dart2_fix.dart]: Oh I see, thank you :)

**Comment by devoncarew** [comment]
And, you likely need an `import 'dart:async';` now (https://travis-ci.org/dart-lang/dart2_fix/builds/383416131#L572), to reference the `Future`.

**Comment by bcko** [comment]
Thank you for your help! :)

**Comment by devoncarew** [comment]
Thanks!

**Comment by eernstg** [comment]
Forgive me for commenting on an already closed PR, but here's a bit of information that may be useful in similar situations:

The language team actually decided to allow an async method to have the return type `void`, and the motivation is that this is a meaningful representation of the intent that invocations of that function should be a "fire and forget" operation: Callers should discard the returned future, and this means that they can't even synchronize on the completion of the associated task.

Considering an async `main`, we actually have a good match for this description. The specification says nothing about how the returned value is used, and there is no support for claiming that the returned value is not ignored.

This means that we could make the case that `void main() async {...}` is fine, and in the concrete context of this PR it also allows us to avoid an import.

PS: This is new, so the available tools may or may not have the feature implemented in any given context.

**Comment by devoncarew** [comment]
Awesome, thanks for the context and explanation @eernstg!

**Pull request merged**